worlds/pokemon_rb/locations.py

```diff
@@ -3,8 +3,34 @@
 from .rom_addresses import rom_addresses
 loc_id_start = 172000000
 
+
+def trainersanity(multiworld, player):
+    return multiworld.trainersanity[player].value == 1
```

Comment:
![image](https://user-images.githubusercontent.com/3189725/206027494-f43548e0-fed2-4b01-9dfa-b412dbdeb591.png)
```suggestion
    return multiworld.trainersanity[player]
```